ubtue_DeGruyter.js

```diff
@@ -0,0 +1,85 @@
+{
+	"translatorID": "9ef1752e-bd32-49bb-9d9b-f06c039712ab",
+	"label": "ubtue_DeGruyter",
```

Comment:
Should this translator replace the current DeGruyter.js?

Reply:
yes